wiki/markdown/service-usage.md

```diff
@@ -0,0 +1,85 @@
+| /template/{type}/{mp} | GET    | type - Eiffel event type mp - message protocol, required                                                                                    |                                                                                                                                                        | This endpoint is used to obtain Eiffel event templates implemented in  [Eiffel REMReM Semantics](https://github.com/eiffel-community/eiffel-remrem-semantics).                         |
+| /versions             | GET    |                                                                                                                                             |                                                                                                                                                        | This endpoint is used to get versions of generate service and all loaded protocols versions  in JSON format.                                                                           |
+
+
+## Examples
+
+Typical examples of usage Eiffel REMReM Generate Service endpoints are described below.
+
+You can use command line tools like [curl](https://curl.haxx.se/) or some plugin for your favorite browser. For example:
+
+*   [Postman](https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop) for Chromium-based browsers
+*   [HttpRequester](https://addons.mozilla.org/en-US/firefox/addon/httprequester/) for Firefox
+
+### Examples for `/mp` endpoint
+
+#### Given one message in file _body-single.json_:
+
+```
+curl -XPOST -H "Content-Type: application/json" --data @body-single.json http://localhost:8080/generate-service/eiffelsemantics?msgType=eiffelactivityfinished
+```
+
+Result:
+
+```
+{"meta":{"id":"29f9b8b1-9d4b-41cf-9f42-4e6157850757","type":"EiffelActivityFinishedEvent","version":"3.0.0","time":1499076743638,"tags":["tag1","tag2"],"source":{"domainId":"eiffellocal","host":"01HW758546","name":"fem-eiffellocal","serializer":"pkg:maven/com.github.eiffel-community/eiffel-remrem-semantics@2.0.4","uri":https://fem101-eiffel006.lmera.ericsson.se:8443/jenkins/job/FemBuilder/1100/"}}"data":{"outcome":{"conclusion":"SUCCESSFUL","description":"The Result was Successful"},"persistentLogs":[],"customData":[{"key":"translatedSourceID","value":"29f9b8b1-9d4b-41cf-9f42-4e6157850757"}]},"links":[{"type":"ACTIVITY_EXECUTION","target":"69538470-4c3f-4c16-b84e-64ff0275f6fc"},{"type":"FLOW_CONTEXT","target":"ea82a50c-53bb-4d0c-8f03-b50fb83feb16"}]}
```

Comment:
The example data in these events should be updated. Remove all traces of fem. And update the uri to something more generic.

Reply:
Hi Emelie,

Updated the suggested changes